ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/monitor_connectivity.py

```diff
@@ -34,28 +34,33 @@ def monitor_connectivity(self) -> dict | None:
         for policy in self._filtered_internet_exit_policies:
             for connection in policy["connections"]:
                 if connection["type"] == "tunnel":
-                    interface_sets.append(
-                        {
-                            "name": f"SET-Tunnel{connection['tunnel_id']}",
-                            "interfaces": f"Tunnel{connection['tunnel_id']}",
-                        }
-                    )
+                    interfaceName = f"Tunnel{connection['tunnel_id']}"
```

Comment:
We don't use Camel case in AVD so let's keep it that way
```suggestion
                    interface_name = f"Tunnel{connection['tunnel_id']}"
```